patches/0050-Normalize-retentionDays-in-aws_controltower_landing_.patch

```diff
@@ -151,7 +151,7 @@ index 0000000000..a8bb57939e
 +	actual, err := resourceLandingZoneNormalizeManifest(`
 +	{
 +	  "governedRegions": [
-+	    "ap-southeast-2"
++	    "REGION"
```

Comment:
Does the upstream linter catch hardcoded regions/partitions?

Reply:
Yep!